### PR TITLE
Replace the old detail view with a tabular layout

### PR DESCRIPTION
**Pull request by mehalshah**
Testing isn't ready yet but thought I'd put this out there now

Replace the old detail view with a series of tables. Items that can be scored have a scoring column on the right

### CODE REVIEW

File: apps/src/code-studio/pd/application_dashboard/detail_view_contents.jsx [hunk 1]
```diff
@@ -759,6 +592,205 @@ export class DetailViewContents extends React.Component {
...
+            please click “Send email” to the principal asking for approval.
+          </p>
+          <PrincipalApprovalButtons
+            applicationId={this.props.applicationId}
+            showSendEmailButton={true}
+            onChange={this.handlePrincipalApprovalChange}
+          />
+        </div>
+      );
+    } else { // incomplete
```
Comment (clareconstantine) [inline apps/src/code-studio/pd/application_dashboard/detail_view_contents.jsx]: just calling out that this part may still be incomplete - I know you're already tracking this.

File: apps/src/code-studio/pd/application_dashboard/principal_approval_buttons.jsx [hunk 2]
```diff
@@ -52,6 +54,10 @@ export default class PrincipalApprovalButtons extends React.Component {
         this.props.applicationId,
         data.principal_approval
       );
+      this.setState({
+        sendEmailRequest: null,
+        showSendEmailButton: false
```
Comment (clareconstantine) [inline apps/src/code-studio/pd/application_dashboard/principal_approval_buttons.jsx:59]: would we still want to show this button so they can resend if desired?

**Review by clareconstantine — APPROVED** [review]

File: apps/src/code-studio/pd/application_dashboard/detail_view_contents.jsx (hunk 1 above)
Comment (mehalshah) [reply apps/src/code-studio/pd/application_dashboard/detail_view_contents.jsx]: Sorry - this means that the principal approval has been sent but not completed

File: apps/src/code-studio/pd/application_dashboard/principal_approval_buttons.jsx (hunk 2 above)
Comment (mehalshah) [reply apps/src/code-studio/pd/application_dashboard/principal_approval_buttons.jsx:59]: I think we shouldn't but lets see how people like it in the bug bash

**Pull request merged**